### PR TITLE
fix docs stating to put null in an int.

### PR DESCRIPTION
**Pull request by driesdeproost**

### CODE REVIEW

File: alfresco-solrapi-client/alfresco-solrapi-client-spi/src/main/java/eu/xenit/alfresco/solrapi/client/spi/SolrApiClient.java [hunk 1]
```diff
@@ -92,7 +92,7 @@ AclChangeSets getAclChangeSets(Long fromCommitTime, Long minAclChangeSetId, Long
      * @param minTxnId minimum transaction id (inclusive) - can be {@code null}
      * @param toCommitTime commit time in ms exclusive upper bound restriction - can be {@code null}
      * @param maxTxnId maximum transaction id (exclusive) - can be {@code null}
-     * @param maxResults maximum number of results - can be {@code null}
+     * @param maxResults maximum number of results - use {@code 0} or {@code Integer.MAX_VALUE} in order to omit
```
Comment (tgeens) [inline alfresco-solrapi-client/alfresco-solrapi-client-spi/src/main/java/eu/xenit/alfresco/solrapi/client/spi/SolrApiClient.java:95]: Current impl:
```
conditionalQueryParam(uriBuilder, "maxResults", maxResults, val -> val != null && val != Integer.valueOf(0) && val != Integer.valueOf(Integer.MAX_VALUE));
```
But the backend impl seems to default to 1024 when `maxResults == null` ?

Comment (kerkhofsd) [reply alfresco-solrapi-client/alfresco-solrapi-client-spi/src/main/java/eu/xenit/alfresco/solrapi/client/spi/SolrApiClient.java:95]: Can we process this comment as well?

Comment (driesdeproost) [reply alfresco-solrapi-client/alfresco-solrapi-client-spi/src/main/java/eu/xenit/alfresco/solrapi/client/spi/SolrApiClient.java:95]: I don't see the default behavior that is referred to. Maybe it was already changed?

Comment (driesdeproost) [reply alfresco-solrapi-client/alfresco-solrapi-client-spi/src/main/java/eu/xenit/alfresco/solrapi/client/spi/SolrApiClient.java:95]: `maxResults == null` is illegal for maxResults, a primitive int

**Pull request merged**